mpas_analysis/config.default

```diff
@@ -142,7 +142,7 @@ htmlSubdirectory = html
 # option:
 #    ./run_mpas_analysis config.analysis --generate \
 #         only_ocean,no_timeSeries,timeSeriesSST
-generate = ['all_publicObs', 'no_icebergs']
+generate = ['all_publicObs']
```

Comment:
@kevinrosa, I think this probably got changed as part of fixing up a rebase?  I think we do want the `'no_icebergs'` to be the default.